src/components/custom/_simple-steps.tsx

```diff
@@ -1,18 +1,16 @@
-import React, { ReactElement, ReactNode } from 'react'
+import React from 'react'
 import styled from 'styled-components'
 import Button from './_button'
-import { localize } from 'components/localization'
 import { Header, Text } from 'components/elements'
-import { Container, SectionContainer, Flex, Desktop, Mobile } from 'components/containers'
+import { Container, SectionContainer, Flex } from 'components/containers'
 import device from 'themes/device'
 import useHandleSignup from 'components/hooks/use-handle-signup'
-import useAuthCheck from 'components/hooks/use-auth-check'
-import { usePlatformQueryParam } from 'components/hooks/use-platform-query-param'
+import { TString } from 'types/generics'
+import { Localize } from 'components/localization'
 
 type SimpleStepsProps = {
-    content?: { header?: ReactNode; icon?: HTMLImageElement; text?: ReactElement }[]
-    header?: string
-    sign_up?: boolean
+    content?: { header?: TString; icon?: React.ReactElement; text?: TString }[]
```

Comment:
possible to  put this typescript under a separate variable name and call it here 
```suggestion
    content?:  contentType[]
```